api/src/paths/dwc/elastic-search.ts

```diff
@@ -0,0 +1,137 @@
+import { RequestHandler } from 'express';
```

Comment:
I would change the name of this file, as elastic search isn't a detail the outside world needs to know about.

Either something like `paths/dwc/eml/search` or `paths/dwc/search

Not 100% certain how the search endpoints will be structured yet (multiple endpoints vs a single endpoint), so either is probably fine for now and we can adjust if needed in future.

Reply:
They just care what the endpoint will do for them (performing some kind of search function on dwc data).